admin-frontend/src/pages/Status.js

```diff
@@ -247,4 +309,4 @@ class Status extends React.Component {
   }
 }
 
-export default Status;
+export default Status;
```

Comment:
did you run Prettier on the code?

Reply:
No. I don't know if you use prettier in this project, but I wasn't using.